### DISCUSSION TITLE
Gridtracker Connection -Invalid Response - Wrong JSON

### DISCUSSION THREAD

**Discussion by itsthemedication** [magicbug/Cloudlog#2848] [Third Party Tools]
OS: Linux Mint 21
Cloudlog: v2.6.0
------------------------------------------------------------
1. I create a logbook and find the id - it's 1
2. I create an API key
3. I insert the following info in GridTracker:

url:          http://192.168.1.102/index.php/api/qso 
api:         copy and paste from cloudlog api page
logbook: 1

Gridtracker gives an "invalid response"
Cloudlog displays:

status	"failed"
reason	"wrong JSON"
 
Thanks in advance.

**Comment by magicbug** [comment]
I'd make sure the URL is correct I doubt gridtracker needs the index.php stuff

**Reply by itsthemedication** [reply]
Made no difference. Thanks for responding.

**Comment by OM-Bort** [comment]
I have the same problem here.
Are there any other suggested solutions?
